pkg/tsdb/azuremonitor/azure-log-analytics-datasource.go

```diff
@@ -163,9 +169,6 @@ func (e *AzureLogAnalyticsDatasource) executeQuery(ctx context.Context, query *A
 	azlog.Debug("AzureLogAnalytics", "Request ApiURL", req.URL.String())
 	res, err := ctxhttp.Do(ctx, dsInfo.Services[azureLogAnalytics].HTTPClient, req)
 	if err != nil {
-		if !dsInfo.Settings.AzureLogAnalyticsSameAs {
```

Comment:
remove the `AzureLogAnalyticsSameAs` field from `Settings` since you don't use it anymore.
I

Reply:
good point.